### DISCUSSION TITLE
Recents picker like Telescope

### DISCUSSION THREAD

**Discussion by txtyash** [helix-editor/helix#3809] [Q&A] [answered]
Is there a recently opened file picker in helix? I couldn't find one.

**Comment by the-mikedavis** [comment]
<kbd>Space</kbd><kbd>'</kbd> opens the most recently used picker

**Reply by txtyash** [reply]
![recents_picker](https://user-images.githubusercontent.com/85027668/189605783-323c2bdc-418f-4ad8-b58a-bf076dc6cdd3.png)

This is how a recents picker looks like in LunarVim. It lists all the files I'd ever opened.

**Reply by the-mikedavis** [reply]
Oh, I read the post wrong. This is not implemented. In order to work across sessions it would need https://github.com/helix-editor/helix/issues/401

**Comment by txtyash** [comment, marked as answer]
#3812
